src/main/app/src/app/admin/model/case-model.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import {PlanItemDefinitie} from './plan-item-definitie';
+
+export class CaseModel {
```

Comment:
Ik zie net dat in Flowale dit een CaseDefinition heet (zie org.flowable.cmmn.api.repository.CaseDefinition). We kunnen deze terminology beter gelijk houden